### ISSUE TITLE
Image size not working on web

### ISSUE THREAD

**Issue by mathis-kdio** [gluestack/gluestack-ui#1429]
### Description

The "size" property doesn't seem to work on the web with expo

### Steps to reproduce

2. Click on '...' 
3. Scroll down to '...' 

### gluestack-ui Version

1.0.19

### Platform

- [X] Expo
- [ ] React Native CLI
- [ ] Next
- [X] Web
- [ ] Android
- [ ] iOS

### Additional Information

```
  <VStack alignItems='center'>
    <Image
      size='xl'
      alt="Logo de l'application"
      source={require('@assets/icon.png')}/>
  </VStack>
```

![image](https://github.com/gluestack/gluestack-ui/assets/11821148/cdc9f317-fb3c-4019-93be-0de20725827a)

**Comment by makkarMeenu** [comment]
Thanks for reporting, we will have a look!

**Comment by makkarMeenu** [comment]
I couldn't reproduce this in expo application. Can you please provide me an expo repository with error producible code?

**Comment by mathis-kdio** [comment]
[here](https://snack.expo.dev/@matkdio/image-web-size) an example. If you change xl to sm, the image doesn't change size on the web, but it does on Android.

**Comment by jayala** [comment]
Is it possible that this is caused by the project using React Native's Image component instead of [Expo's Image](https://docs.expo.dev/versions/latest/sdk/image/)? https://github.com/gluestack/gluestack-ui/blob/main/packages/themed/src/components/Image/styled-components/Root.tsx#L1

**Comment by surajahmed** [comment]
This issue is fixed in `@gluestack-ui/config` version `1.0.12`. Please update it to the latest and try again. 
Feel free to reopen if this issue persists.
